validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -147,13 +151,97 @@ func (grpcValidatorClient) GetAggregatedSyncSelections(context.Context, []iface.
 }
 
 func NewGrpcValidatorClient(cc grpc.ClientConnInterface) iface.ValidatorClient {
-	return &grpcValidatorClient{ethpb.NewBeaconNodeValidatorClient(cc)}
-}
+	return &grpcValidatorClient{ethpb.NewBeaconNodeValidatorClient(cc), false}
+}
+
+func (c *grpcValidatorClient) StartEventStream(ctx context.Context, topics []string, eventsChannel chan<- *eventClient.Event) {
+	ctx, span := trace.StartSpan(ctx, "validator.gRPCClient.StartEventStream")
+	defer span.End()
+	if len(topics) == 0 {
+		eventsChannel <- &eventClient.Event{
+			EventType: eventClient.EventError,
+			Data:      []byte(errors.New("no topics were added").Error()),
+		}
+		return
+	}
+	// TODO: ONLY WORKS WITH HEAD TOPIC RIGHT NOW/ONLY PROVIDES THE SLOT
```

Comment:
Why are you yelling? (all caps) :rofl:

Reply:
big reminder that this is ducktaped together lol it's still using the old stream slots